packages/components/card/src/Card.tsx

```diff
@@ -37,8 +38,21 @@ export default defineComponent({
       return children.value.some(node => node.type && (node.type as any).name === 'IxCardGrid')
     })
 
+    const [selected, setSelected] = useControlledProp(props, 'selected', () => false)
+    const isDisabled = computed(() => props.disabled ?? false)
+    const isSelectable = computed(() => props.selectable ?? false)
+    const handleClick = () => {
+      if (isDisabled.value || !isSelectable.value) {
+        return
+      }
+
+      setSelected(!selected.value)
+      callEmit(props.onSelectedChange, !selected.value)
```

Comment:
```suggestion
      const newSelected = !selected.value
      setSelected(newSelected)
      callEmit(props.onSelectedChange, newSelected)
```
这里一定要用一个临时变量保存下，否则 setSelected 后，selected.value 的值就已经发生改变了。